test/_unit.js

```diff
@@ -479,6 +479,8 @@ suite('Lexer', () => {
     const { exports } = parse(`
       module.exports = { 'ab cd': foo };
       exports['not identifier'] = 'asdf';
+      exports['\u{D83C}\u{DF10}'] = 1;
+      exports['\u{D83C}'] = 1;
```

Comment:
Can you check/test how it behaves with escaped quotes? `"\""`/`'\''`

Reply:
This actually caught a bug as well!